doc/rbd/iscsi-overview.rst

```diff
@@ -43,6 +42,11 @@ Network (SAN).
                              |   +-----------+   |
                              +-------------------+
 
+.. warning::
+
+   The iSCSI code is in maintenance as of November 2022. This means that
```

Comment:
```suggestion
   The iSCSI gateway is in maintenance as of November 2022. This means that
```

Reply:
I've made this change. Thanks for your careful edit, Ilya.